src/js/videojs-dash.js

```diff
@@ -89,14 +91,25 @@ class Html5DashJS {
     this.mediaPlayer_.setProtectionData(this.keySystemOptions_);
     this.mediaPlayer_.attachSource(manifestSource);
 
+    // Setup text tracks
+    setupTextTracks.call(null, this.player);
+
+    // When `dashjs` finishes loading metadata, create audio tracks for
+    // `video.js`.
+    this.mediaPlayer_.on(
```

Comment:
minor, any reason why the event handler is bound inline rather than within the setup?